lib/fauxpaas/artifact_builder.rb

```diff
@@ -38,5 +39,36 @@ def add_reference(ref, path)
         .write
     end
 
+		# Set the mode bits on all files in the Artifact
+		# @param path [Pathname]
+		# @param asset_dir [String]
+		def set_access_control(path, asset_dir = "public")
+			asset_path = path/asset_dir 
```

Comment:
Indentation problems -- use 2 spaces. Rubocop can help, but this looks like the product of bad editor settings (`set noexpandtab` and `set ts=8` in place?).